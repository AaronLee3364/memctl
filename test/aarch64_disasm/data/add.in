fffffff0070c0000
910043fd 8b143268 8b2a46ca 9102000a
91400445 310005ce 2b080128 ab2a4109
