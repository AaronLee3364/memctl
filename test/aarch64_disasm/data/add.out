fffffff0070c0000  ADD     X29, SP, #0x10
fffffff0070c0004  ADD     X8, X19, X20, LSL #12
fffffff0070c0008  ADD     X10, X22, W10, UXTW #1
fffffff0070c000c  ADD     X10, X0, #0x80
fffffff0070c0010  ADD     X5, X2, #0x1, LSL #12
